recipes/easymt/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "easymt" %}
+{% set version = "1.0.0" %}
+package:
+  name: {{name}}
+  version: {{version}}
+
+source:
+  url: https://github.com/tang-shuyuan/EeayMT/releases/download/v{{version}}/{{name}}_v{{version}}.tar.gz
+  sha256: d9220c8399eaba7afeae9de7853f3c7a59db47c19669a9101f4b5016750b3d98
+
+build: 
+  number: 0
+  entry_points: 
+    - easymt = extract:main
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('easymt', max_pin="x") }}
+
+requirements: 
+  bulid: 
+    - python >=3.9
+  run: 
+    - blast =2.15.0
```

Comment:
why does it need to be this specifc version?

Reply:
We need to use an output file from BLAST, and we specify a particular version in case the output file format changes. If writing like this is not conducive to building a conda environment, we can make changes.